program/c/src/oracle/util/test_prng_battery.c

```diff
@@ -1,91 +0,0 @@
-#include <stdio.h>
```

Comment:
this file is like a super exhaustive test of the prng (which itself is only used for tests). It's not actually run as part of the existing c test suite, so just get rid of it.